lib/hashes.md

```diff
@@ -5,40 +5,40 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a Hash?
 
-Replace me with your response...
+A Hash is similar to an Array but allows us to organize multiple lists together.
```

Comment:
Hm. I agree this _is_ a useful implementation of a Hash (like you outline in the next question) but it's not the sole reason why we would use a Hash. Feel free to be more general here.